### DISCUSSION TITLE
Changing the default video bitrate and resolution for the sample page

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5337] [FAQs]
I'm publishing a webrtc stream using the webrtc sample publish page index.html
Upon observing the stats on the publish page, I found that the maximum video bitrate is set to 1200 Kbps while the video resolution is set to 480p.

Can you please tell me how and where I can change/set the video bitrate and resolution values?
I'm using AMS EE v2.6.2

**Comment by zerostack0** [comment]
Thanks for your question.

To be able to set the maximum video bitrate, you had 2 options. 

If you wanted to set maximum video bitrate for a specific stream, you can use the [changeBandwidth function](https://github.com/ant-media/StreamApp/blob/v2.5.3/src/main/webapp/samples/publish_webrtc.html#L265) like this `webRTCAdaptor.changeBandwidth(<maxVideoBitrateKbps>, <streamId>);`

If you wanted to set maximum video bitrate for all streams, you can set the bandwidth value on [WebRTCAdaptor's constructor](https://github.com/ant-media/StreamApp/blob/v2.5.3/src/main/webapp/samples/publish_webrtc.html#L529)

Note: Instead of a limit, if you wanted to send the highest quality your network permits, you can set **maxVideoBitrateKbps** value **unlimited**

**Comment by zerostack0** [comment]
We don't have video resolution set on the SDK side because WebRTC protocol tries to send maximum resolution as long as your client-side network handles it. But you can enable the Adaptive Streaming feature to transcode stream into a specific resolution.
